src/Robot.elm

```diff
@@ -28,62 +25,40 @@ subscriptions : Model -> Sub Msg
 subscriptions model =
     Browser.Events.onKeyDown (Json.Decode.map KeyDown Html.Events.keyCode)
 
-update : Msg -> Model -> ( Model, Cmd Msg )
+update : Msg -> Model -> Model
 update msg model =
     case msg of
         KeyDown code ->
             case code of
                 37 ->
-                    ( { model | xblue = model.xblue - 1 }, Cmd.none )
+                    { model | x = model.x - 1 }
 
                 38 ->
-                    ( { model | yblue = model.yblue - 1 }, Cmd.none )
+                    { model | y = model.y - 1 }
 
                 39 ->
-                    ( { model | xblue = model.xblue + 1 }, Cmd.none )
+                    { model | x = model.x + 1 }
 
                 40 ->
-                    ( { model | yblue = model.yblue + 1 }, Cmd.none )
-
-                65 ->
-                    ( { model | xred = model.xred - 1 }, Cmd.none )
-
-                87 ->
-                    ( { model | yred = model.yred - 1 }, Cmd.none )
-
-                68 ->
-                    ( { model | xred = model.xred + 1 }, Cmd.none )
-
-                83 ->
-                    ( { model | yred = model.yred + 1 }, Cmd.none )
+                    { model | y = model.y + 1 }

```

Comment:
What if you want to move faster?
4 `1`s are a lot to change for such a common task...